_data/actions/269.yml

```diff
@@ -19,3 +19,5 @@ desc: |-
       UseItem("Potn52",Myself)
   END
   ```
+
+  If creature name if unset (strref=`-1`), it will be attributed to the protagonist in the message log. To avoid that, use `DisplayStringNoNameHead()`, or `SAY` creature's `NAME` to ~~ (empty string).
```

Comment:
```suggestion
  If the object's name is unset or set to an invalid strref, it will be attributed to the protagonist in the message log. To avoid that, use `DisplayStringNoNameHead()`, or `SAY` the creature's `NAME` to ~~ (empty string) when coding it in WeIDu.
```